acl_upgrade.php

```diff
@@ -480,19 +480,66 @@
   updateAcl($doc_write, 'Physicians', 'patients', 'Patients', 'amendment', 'Amendments (write,addonly optional)', 'write');
   //Insert the 'lab' object from the 'patients' section into the Physicians group write ACL (added in 5.0.1)
   updateAcl($doc_write, 'Physicians', 'patients', 'Patients', 'lab', 'Lab Results (write,addonly optional)', 'write');
-
-  //DONE with upgrading to this version
+   //DONE with upgrading to this version
   $acl_version = $upgrade_acl;
 }
 
-/* This is a template for a new revision, when needed
+ //This is a template for a new revision, when needed
 // Upgrade for acl_version 5
 $upgrade_acl = 5;
 if ($acl_version < $upgrade_acl) {
   echo "<B>UPGRADING ACCESS CONTROLS TO VERSION ".$upgrade_acl.":</B></BR>";
 
   //Collect the ACL ID numbers.
   echo "<B>Checking to ensure all the proper ACL(access control list) are present:</B></BR>";
+//Get Accountant ACL ID number
+    $admin_write = getAclIdNumber('Administrators', 'write');
+
+
+  //Add new object Sections
+  echo "<BR/><B>Adding new object sections</B><BR/>";
+    // Add 'Groups' object (added in 5.0.1)
+    addObjectSectionAcl('Groups', 'groups');
```

Comment:
this is flip-flopped. see examples in this script. should be:
``````
addObjectSectionAcl('groups', 'Groups');